<!DOCTYPE HTML>
<!--
	Miniport by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>Publications | Yuanning-Li Lab</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" href="pics/icon_home.png">
	</head>
	<body style="background-color:#fffcfc">

		<!-- Nav -->
		<nav id="nav">
			<ul class="container" >
			<!--
			<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#d16f2a" size="6.8">The Shen Laboratory</font></p>
			-->
				<p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 20px"> <font face="verdana" color="#ffffff" size="6.8">The Yuanning-Li Laboratory</font></p>
		    <p style="text-indent;margin-left;margin-bottom: 2px;margin-top: 10px"> <font face="verdana" color="#ffffff" size="5">Phylogenomics & Genomics in Marine Organisms</font></p>
			</ul>

			<ul class="container">
			
					<li><a href="index.html"><font face="verdana" color="#ffffff" size="4.8">HOME</font></a></li>
					<!--li><a href="about.html"><font face="verdana" color="#ffffff" size="4.8">ABOUT</font></a></li-->
				    <li><a href="research.html"><font face="verdana" color="#ffffff" size="4.8">RESEARCH</font></a></li>
					<li><a href="people.html"><font face="verdana" color="#ffffff" size="4.8">PEOPLE</font></a></li>
					<li><a href="publications.html"><font face="verdana" color="#d16f2a" size="4.8">PUBLICATIONS</font></a></li>
					<li><a href="contact.html"><font face="verdana" color="#ffffff" size="4.8">CONTACT</font></a></li>
			</ul>
		</nav>

		
		<!-- almetric script -->
		<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>


		<!-- Publications -->
		<div class="container">
			<article id="PUBLICATIONS">
				<header>
				    <br></br>
					<br>
					<p style="margin-bottom: 2px;margin-top: 150px"><strong>Citations are available at <a target="_blank" href="https://scholar.google.com/citations?user=65ygCIsAAAAJ&hl=en">Google Scholar</a>.</br> Lab members are indicated in <b><font face="verdana">bold</font></b>. # co-first author, * corresponding author.<strong></p>
	                <br>

                  <script>
                   var Now = new Date(),
				   Timezone = Now.getTimezoneOffset()/60, //GMT 时间与本地时间以分钟计的时间差, 换成小时； nashville：UTC-5h; China: UTC +8
				   //The time-zone offset is the difference, in minutes, between UTC and local time. Note that this means that the offset is positive if the local timezone is behind UTC and negative if it is ahead. For example, if your time zone is UTC+10 (Australian Eastern Standard Time), -600 will be returned. Daylight savings time prevents this value from being a constant even for a given locale
				   
				   CurrentDay = Now.getDay(),
                   OpeningTime = new Date(Now.getFullYear(), Now.getMonth(), Now.getDate(), 22, 00),
                   ClosingTime = new Date(Now.getFullYear(), Now.getMonth(), Now.getDate(), 7, 00),
                   Open = (Now.getTime() > OpeningTime.getTime() && Now.getTime() < ClosingTime.getTime());
				   //document.write('Today is ');
				   //document.write(Now);
				   //document.write('<br>');
				   //document.write('<br>');
				    //document.write(Timezone);
				    //document.write('<br>');
				    //document.write(CurrentDay);
				    //document.write('<br>');
				    //document.write(Now.getFullYear());
				    //document.write('<br>');
				    //document.write(Now.getMonth());
				    //document.write('<br>');
				    //document.write(Now.getDay());
				    //document.write('<br>');
				    //document.write(Now.getTime());
				    //document.write('<br>');
				    //document.write(OpeningTime.getTime());
				    //document.write('<br>');
				    // document.write(ClosingTime.getTime());
				    //document.write('<br>');
                    // days 0.sun 1.mon 2.tues 3.wed 4.thur 5.fri 6.sat 
                    // if (CurrentDay !== 6 && CurrentDay !== 0 && Open) {
					// if (Timezone == -8 && Open) {
				    //if (Timezone == -8 ) { // only display in China
                    //  document.write('<p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">16. Kominek J.*, Doering D.T.*, Opulente D., <b><font face="verdana">Shen X.-X.</font></b>, Zhou X., Jeremy D., Hulfachor A.B., Mcgee M., Karlen S., Kurtzman C.P., Rokas A., Hittinger C.T. 2019. Eukaryotic acquisition of a bacterial operon. <b><font face="verdana"><i>Cell</i></font></b> in press. (*co-first author)<div style="padding-left: 1em;"></div></p>');
                   //   }
				   // </script>

				   	 <p style="margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">Preprints/Submitted articles</font></b>
					   <hr style="width:100%;text-align:left;margin-bottom: -2px;margin-top: 20px">
					 </p>


                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                         
                      1. <b><font face="verdana">Li Y.</font></b>, J.L. Steenwyk, A.L. LaBella, M.C. Harrison, M. Groenewald, X. Zhou, X.-X. Shen, T. Zhao, C.T. Hittinger, A. Rokas (2022). Contrasting modes of macro- and micro-synteny evolution in a eukaryotic subphylum. Submitted.
					  </p>
                      </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                         
                      2.Steenwyk, J. L.*, D. C. Goltz, T. J. Buida III, <b><font face="verdana">Y. Li</font></b>, X.-X. Shen, & A. Rokas*. orthoSNAP: a tree splitting and pruning algorithm for retrieving single-copy orthologs from gene family trees. <b><font face="verdana"><i>bioRxiv</i></font></b>. <a target="_blank" href="https://www.biorxiv.org/content/10.1101/2021.10.30.466607v1">bioRxiv preprint.</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1101/2021.10.30.466607" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1101/2021.10.30.466607" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
                      

             
					 <p style="margin-bottom: 2px;margin-top: 2px">
                       <b><font face="verdana" color="#d16f2a" size="5">Research articles</font></b>
					   <hr style="width:100%;text-align:left;margin-bottom: -2px;margin-top: 20px">
					 </p>
					
                     <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                         
                    
                      26.Steenwyk, J. L.*, T. J. Buida III, C. Goncalves, D. C. Goltz, G. H. Morales, M. E. Mead, A. L. LaBella, C. M. Chavez, J. E. Schmitz, M. Hadjifrangiskou, <b><font face="verdana">Y. Li</font></b>, & A. Rokas*.  BioKIT: a versatile toolkit for processing and analyzing diverse types of sequence data. <b><font face="verdana"><i>Genetics</i></font></b> 221(3):iyac079. <a target="_blank" href="pdfs/2022_Steenwyk_etal_Genetics.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/genetics/iyac079" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/genetics/iyac079" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                     25. Liu D., Chen J., Wang Y., Meng Y., <b><font face="verdana">Li Y.</font></b>, Huang R., Xia Y., Liu H., Jiao N., Xun L., Liu J.* 2022. <i>Synechococcus sp. PCC7002</i> Uses Peroxiredoxin to Cope with Reactive Sulfur Species Stress. <b><font face="verdana"><i>Mbio</i></font></b> e01039-22. (*corresponding author). <a target="_blank" href="pdfs/2022_Liu_etal_mBio.pdf">PDF</a><br/>
                     <div style="padding-left: 1em;">
                     </div>
				     </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      24. <b><font face="verdana">Li Y.</font></b>, Shen X.-X., Evans B., Dunn C.W.*, Rokas A.* 2021. Rooting the animal tree of life. <b><font face="verdana"><i>Molecular Biology and Evolution</i></font></b> 38: 4322–4333. (*corresponding author). <a target="_blank" href="pdfs/2021_Li_etal_MBE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/molbev/msab170" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/molbev/msab170" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       23. <b><font face="verdana">Li Y.</font></b>, Steenwyk J., Chang Y., Wang Y., James T.Y., Stajich J.E., Spatafora J.W., Groenewald M., Dunn C., Hittinger C.T., Shen X.-X.*, Rokas A.* 2021. A genome-scale phylogeny of the kingdom Fungi. <b><font face="verdana"><i>Current Biology</i></font></b> 31: 1653–1665. (*corresponding author). <a target="_blank" href="pdfs/2021_Li_etal_CB.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Scripts_and_analyses_used_for_the_fungal_phylogeny/12751736">Download phylogenomic data</a>, <a target="_blank" href="https://doi.org/10.5281/zenodo.3970286">genome data.</a><br/>
	                  <div style="padding-left: 1em;">
					  <a target="_blank" href="pics/li_CB_cover.tif.jpg">Cover article</a>
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.cub.2021.01.074" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.cub.2021.01.074" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
			         <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      22. Novitsky V., Steingrimsson J., Howison M., Dunn C., Gillani, F.S. Manne, A., <b><font face="verdana">Li Y.</font></b>, Spence M. Parillo, Z., Fulton J. and Marak T., 2021. Longitudinal typing of molecular HIV clusters in a statewide epidemic. <b><font face="verdana"><i>AIDS</i></font></b> 35(11):1711-1722.
					 </p>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                       21. Steenwyk, J. L., M. E. Mead, P. A. Castro, C. Valero, A. Damasio, R. A. C. Santos, A. L. LaBella, <b><font face="verdana">Y. Li</font></b>, S. L. Knowles, H. A. Raja, N. H. Oberlies, X. Zhou, O. A. Cornely, F. Fuchs, P. Koehler*, G. H. Goldman*, & A. Rokas*, 2021. Genomic and phenotypic analysis of COVID-19-associated pulmonary aspergillosis isolates of <i>Aspergillus fumigatus</i>. <b><font face="verdana"><i>Microbiology Spectrum</i></font></b> 9(1): e00010-21. <a target="_blank" href="pdf_new/2021_Steenwyk_etal_MicrobiolSpectrum.pdf">PDF</a><br/>
                      <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1128/Spectrum.00010-21 " data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1128/Spectrum.00010-21 " data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>



                     20. Steenwyk J., Buida III T.J., LaBella A.L., <b><font face="verdana"> Li Y.</font></b>, Shen X.-X., Rokas A. 2021. PhyKIT: a UNIX shell toolkit for processing and analyzing phylogenomic data. <b><font face="verdana"><i>Bioinformatics</i></font></b> 37(16):2325–2331. <a target="_blank" href="pdfs/2021_Steenwyk_etal_PhyKIT_Bioinf.pdf">PDF</a> <a target="_blank" href="https://jlsteenwyk.com/PhyKIT/">Download PhyKIT</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/bioinformatics/btab096" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/bioinformatics/btab096" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      19. Shen X.-X.*, <b><font face="verdana">Li Y.</font></b>,  Hittinger C., Chen X., Rokas A.* 2020. An investigation of irreproducibility in maximum likelihood phylogenetic inference. <b><font face="verdana"><i>Nature Communications</i></font></b> 11:6096 (*corresponding author). <a target="_blank" href="pdfs/2020_Shen_etal_Nat_Comm.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Irreproducibility_of_maximum_likelihood_phylogenetic_inference/11917770">Download data</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <a target="_blank" href="https://natureecoevocommunity.nature.com/posts/irreproducibility-in-phylogenetic-inference">The story behind the paper in Nature Ecology & Evolution blog</a>
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41467-020-20005-6" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41467-020-20005-6" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      18. Steenwyk J., Buida III T.J., <b><font face="verdana">Li Y.</font></b>, Shen X.-X., Rokas A. 2020. ClipKIT: a multiple sequence alignment-trimming algorithm for accurate phylogenomic inference. <b><font face="verdana"><i>PLoS Biology</i></font></b> 18: e3001007.  <a target="_blank" href="pdfs/2020_Steenwyk_etal_ClipKIT_PLoSBiol.pdf">PDF</a>  <a target="_blank" href="https://jlsteenwyk.com/ClipKIT/">Download ClipKT</a>.<br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1371/journal.pbio.3001007" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1371/journal.pbio.3001007" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>
					 </p>
					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
             
					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      17. <b><font face="verdana">Li Y.</font></b>#, David K.T.#, Shen X.-X., Steenwyk J., Halanych K.M., Rokas A. 2020. Feature Frequency Profile-based phylogenies are inaccurate. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> in press (#co-first author). <a target="_blank" href="pdfs/2020_Li_etal_PNAS.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1073/pnas.2013143117" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1073/pnas.2013143117" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

					  <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      16. Shen X.-X.*, Steenwyk J., LaBella A., Opulente D., Zhou X., Kominek J., <b><font face="verdana">Li Y.</font></b>, Groenewald M., Hittinger C., Rokas A.* 2020. Genome-scale phylogeny and contrasting modes of genome evolution in the fungal phylum Ascomycota. <b><font face="verdana"><i>Science Advances </i></font></b> 6(45):eabd0079 (*corresponding author). <a target="_blank" href="pdfs/2020_Shen_etal_Sci_adv.pdf">PDF</a> <a target="_blank" href="https://figshare.com/articles/dataset/Phylogenomics_and_contrasting_modes_of_genome_evolution_in_Ascomycota/12196149">Download phylogenomic data</a>, <a target="_blank" href="https://doi.org/10.5281/zenodo.3783970">genome assemblies and annotations.</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1126/sciadv.abd0079" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1126/sciadv.abd0079" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      15. Novitsky V., Steingrimsson J.A., Howison M., Gillani F.S., <b><font face="verdana">Li Y.</font></b>, Manne A., Fulton J., Spence M., Parillo Z., Marak T. and Chan P.A., 2020. Empirical comparison of analytical approaches for identifying molecular HIV-1 clusters. <b><font face="verdana"><i>Scientific Reports</i></font></b> 10(1):1-11. <a target="_blank" href="pdfs/2020_Shen_etal_Sci_adv.pdf">PDF</a> 
                      <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-020-75560-1" data-hide-zero-citations="true" data-style="large_rectangle"></span></div>
					 </p>

				      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      14. Kantor R., Fulton  J.P., Steingrimsson J., Novitsky V., Howison M., Gillani F., <b><font face="verdana">Li Y.</font></b>, Manne A., Parillo Z., Spence M. and Marak T., 2020. Challenges in evaluating the use of viral sequence data to identify HIV transmission networks for public health. <b><font face="verdana"><i>Statistical Communications in Infectious Diseases</i></font></b> 12(S1). <a target="_blank" href="pdf_new/2020_kantor_etal.pdf">PDF</a>
					  </p>
                    </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      13. Kong L.#, <b><font face="verdana">Li Y.#</font></b>, Kocot K.M., Yang Y., Qi L., Li Q. and Halanych K.M.*, 2020. Mitogenomics reveals phylogenetic relationships of Arcoida (Mollusca, Bivalvia) and multiple independent expansions and contractions in mitochondrial genome size. <b><font face="verdana"><i>Molecular Phylogenetics and Evolution</i></font></b> 150, p.106857(#co-first author). <a target="_blank" href="pdf_new/2020_Kong_etal_MPE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2020.106857" data-hide-zero-citations="true" data-style="large_rectangle"></span>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      12. Zhou Y.*, <b><font face="verdana">Li Y.</font></b>, Cheng H., Halanych K.M. and Wang C.*, 2020. The mitochondrial genome of the bone-eating worm Osedax rubiplumus (Annelida, Siboglinidae). <b><font face="verdana"><i>Mitochondrial DNA Part B</i></font></b> 5(3):2267-2268. <a target="_blank" href="pdf_new/2020_Zhou_etal_MitochondrialpartB.pdf">PDF</a><br/>
					  </p>
					 </p>
                
                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      11. Zhou Y., Wang, <b><font face="verdana">Y. Li</font></b>, Y., Shen C., Liu Z. and Wang C., 2020. First report of Osedax in the Indian Ocean indicative of trans-oceanic dispersal through the Southern Ocean. <b><font face="verdana"><i>Marine Biodiversity</i></font></b> 50(1):1-9. <a target="_blank" href="pdf_new/2020_Zhou_etal_MarineBiodiversity.pdf">PDF</a><br/>
	            	  </p>
					 </p>

					 <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      10. <b><font face="verdana">Li Y.</font></b>, Tassia M.G., Waits D.S., Bogantes V.E., David K.T. and Halanych K.M., 2019. Genomic adaptations to chemosymbiosis in the deep-sea seep-dwelling tubeworm Lamellibrachia luymesi. <b><font face="verdana"><i>BMC Biology</i></font></b> 17(1):1-14. <a target="_blank" href="pdf_new/2018_Li_etal_BMCBiology.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1186/s12915-019-0713-x" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1186/s12915-019-0713-x" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      9. Voight J.R., Marshall B.A., Judge J., Halanych K.M., <b><font face="verdana">Li Y.</font></b>, Bernardino A.F., Grewe F. and Maddox J.D., 2019. Life in wood: preliminary phylogeny of deep-sea wood-boring bivalves (Xylophagaidae), with descriptions of three new genera and one new species. <b><font face="verdana"><i>Journal of Molluscan Studies</i></font></b> 85(2):1-9. <a target="_blank" href="pdf_new/2019_Voight_etal_MolluscanStudy.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/mollus/eyz003" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/mollus/eyz003" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
                      </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      8. <b><font face="verdana">Li Y.</font></b>*, Kocot K.M., Tassia M.G., Cannon J.T., Bernt M. and Halanych K.M.*, 2019. Mitogenomics reveals a novel genetic code in Hemichordata. <b><font face="verdana"><i>Genome Biology and Evolution</i></font></b> 11(1):29-409. <a target="_blank" href="pdfs/2020_Li_etal_PNAS.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1093/gbe/evy254" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1093/gbe/evy254" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      7. Galaska M.P., <b><font face="verdana">Li Y.</font></b>, Kocot K.M., Mahon A.R. and Halanych K.M.*, 2019. Conservation of mitochondrial genome arrangements in brittle stars (Echinodermata, Ophiuroidea). <b><font face="verdana"><i>Molecular Phylogenetics and Evoluion</i></font></b> 30, pp.115-120. <a target="_blank" href="pdf_new/2019_Galaska_etal_MPE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2018.10.002" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.ympev.2018.10.002" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      6. Li N., Bao L., Zhou T., Yuan Z., Liu S., Dunham R., <b><font face="verdana">Li Y.</font></b>, Wang K., Xu X., Jin Y. and Zeng Q., 2018. Genome sequence of walking catfish (Clarias batrachus) provides insights into terrestrial adaptation. <b><font face="verdana"><i>Proc. Natl. Acad. Sci. USA</i></font></b> 19(1):1-16. <a target="_blank" href="pdf_new/2018_Li_etal_BMCBiology.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1186/s12864-018-5355-9" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1186/s12864-018-5355-9" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      5. <b><font face="verdana">Li Y.*</font></b>, Liles M.R. and Halanych K.M.*, 2018. Endosymbiont genomes yield clues of tubeworm success. <b><font face="verdana"><i>The ISME journal</i></font></b> 12(11):2785-2795. <a target="_blank" href="pdf_new/2018_Li_etal_ISME.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41396-018-0220-z" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41396-018-0220-z" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      4. Bernardino A.F., <b><font face="verdana">Li, Y.</font></b>, Smith C.R. and Halanych K.M., 2017. Multiple introns in a deep-sea Annelid (Decemunciger: Ampharetidae) mitochondrial genome. <b><font face="verdana"><i>Scientific Reports</i></font></b> 7(1):1-8. <a target="_blank" href="pdf_new/2018_Bernadino_ScientificReports.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1038/s41598-017-04094-w" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1038/s41598-017-04094-w" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      3. <b><font face="verdana">Li Y.*</font></b>, Kocot K.M., Whelan N.V., Santos S.R., Waits D.S., Thornhill D.J. and Halanych K.M.*, 2017. Phylogenomics of tubeworms (Siboglinidae, Annelida) and comparative performance of different reconstruction methods. <b><font face="verdana"><i>Zoologica Scripta</i></font></b> 46(2):200-213. <a target="_blank" href="pdf_new/Li_et_al_2016_Siboglinidae_phylogenomics.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1111/zsc.12201" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1111/zsc.12201" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      2. Waits D.S., Santos S.R., Thornhill D.J., <b><font face="verdana">Li Y</font></b>. and Halanych K.M.*, 2016. Evolution of sulfur binding by hemoglobin in Siboglinidae (Annelida) with special reference to bone-eating worms, Osedax. <b><font face="verdana"><i> Journal of molecular evolution</i></font></b> 82(4-5):219-229. <a target="_blank" href="pdf_new/2016_Waits_etal_JournalOfMolecularEvolution.pdf">PDF</a><br/>
					  </p>
					 </p>

                      <p style="text-indent:-1em;margin-left:1em;margin-bottom: 2px;margin-top: 2px">
                      1. <b><font face="verdana">Li Y.</font></b>, Kocot K.M., Schander C., Santos S.R., Thornhill D.J. and Halanych K.M., 2015. Mitogenomics reveals phylogeny and repeated motifs in control regions of the deep-sea family Siboglinidae (Annelida). <b><font face="verdana"><i> Molecular Phylogenetics and Evolution</i></font></b> 85:221-229. <a target="_blank" href="pdf_new/2015_Li_etal_MPE.pdf">PDF</a><br/>
	                  <div style="padding-left: 1em;">
					  <script async src="https://badge.dimensions.ai/badge.js" charset="utf-8"></script>
					  <span class="__dimensions_badge_embed__" data-doi="10.1016/j.ympev.2015.02.008" data-hide-zero-citations="true" data-style="large_rectangle"></span>
			          <div data-badge-type="1" data-doi="10.1016/j.ympev.2015.02.008" data-hide-no-mentions="true" data-hide-less-than="1" data-badge-popover='right' class="altmetric-embed"></div>	
					 </div>
					  </p>
					 </p>

                  
                         
				</header>
					  <hr style="width:100%;text-align:left;margin-left;margin-bottom: -10px;margin-top: 2px">
				 <footer>
                       <p style="font-size:16px;text-align:left;;margin-bottom: -900px;"><b><font face="verdana" color="#d16f2a">&#169 All rights reserved by the Yuanning-Li lab</p>
                  </footer>
			</article>
		</div>





</body>
</html>
